services/flathub/flathub.tester.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const { isVPlusDottedVersionNClauses } = require('../test-validators')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('Flathub (valid)').get('/org.mozilla.firefox.json').expectBadge({
+  label: 'flathub',
+  message: isVPlusDottedVersionNClauses,
+})
+
+t.create('Flathub (valid)')
+  .get('/org.mozilla.firefox.json')
+  .intercept(nock =>
+    nock('https://flathub.org')
+      .get('/api/v1/apps/org.mozilla.firefox')
+      .reply(200, {
+        flatpakAppId: 'org.mozilla.firefox',
+        currentReleaseVersion: '78.0.1',
+      })
+  )
+  .expectBadge({ label: 'flathub', message: 'v78.0.1' })
```

Comment:
Not strictly necessary to have this test, especially given the live test above. I don't feel too strongly about it though so no need to block.